etcd-cluster.yaml

```diff
@@ -53,6 +53,10 @@ Resources:
         FromPort: 2379
         ToPort: 2380
         CidrIp: 0.0.0.0/0
+      - IpProtocol: tcp
+        FromPort: 9100
+        ToPort: 9100
+        CidrIp: 0.0.0.0/0
```

Comment:
That would be another way of doing it. The main argument for putting it here is that by default it is now excluded from monitoring, whereas it is vital that some monitoring takes place.

Within a regular stups vpc, I assume this to be ok. A dynamic setup would also be interesting.